### DISCUSSION TITLE
[console][deploy] Show how many replicas for each microservice are being deployed

### DISCUSSION THREAD

**Discussion by lucas-puerari** [mia-platform/community#122] [Ideas]
Has it ever happened to you that a microservice is visible on the deployment page, but then not to find it on the monitoring page? 
It has to me, and it took me a while to realize that 0 replicas had been set for that service.

Might it be helpful to show this information, alongside each microservice, on the deployment page?

**Comment by fredmaggiowski** [comment]
Hey, I think this can be a nice small addition to the deploy section @albertotessarotto should we track this between the small improvements?

**Reply by lucas-puerari** [reply]
Thanks :)

**Comment by davidebianchi** [comment]
Hi @lucas-puerari, interesting. We could add a column with the replicas, specifying if they are static or with HPA. If static we can show the number of replicas, if hpa min and max

**Reply by lucas-puerari** [reply]
Hi Davide, yes, it could be very useful! Thanks

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
